src/App.tsx

```diff
@@ -1,61 +1,40 @@
 import './App.scss';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import { useState } from 'react';
+import { getUserById } from './services/user';
+import { Todo } from './types/Todo';
+import { PostForm } from './PostForm';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const initialTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
```

Comment:
The `getUserById` function is used here, but if it is asynchronous, it should be handled with `await` or promises. This line assumes it is synchronous, which could lead to issues if `getUserById` is asynchronous.